package.json

```
@@ -0,0 +1,30 @@
+{
+  "type": "module",
+  "devDependencies": {
+    "@eslint/js": "^9.13.0",
+    "@stylistic/eslint-plugin": "^2.9.0",
+    "@stylistic/eslint-plugin-js": "^2.9.0",
+    "eslint": "^9.13.0",
+    "eslint-formatter-checkstyle": "^8.40.0",
+    "eslint-formatter-multiple": "^2.0.0",
+    "globals": "^15.11.0"
+  },
+  "scripts": {
+    "mvnbuild": "",
+    "mvntest": "npm run test",
+    "test": "eslint --color eslint.config.js src/main/resources -f eslint-formatter-multiple"
```

personal preference would be something like this (I always like to have a lint:fix script, you should be able to to do something like `npm run lint -- --fix` but its more to type and remember):
```suggestion
    "mvntest": "npm run lint",
    "lint": "eslint --color eslint.config.js src/main/resources -f eslint-formatter-multiple",
    "lint:fix": "eslint --fix --color eslint.config.js src/main/resources -f eslint-formatter-multiple",
```